api/src/main/java/org/flab/api/domain/event/dto/seat/SeatResponse.java

```diff
@@ -7,11 +7,13 @@
 @AllArgsConstructor
 @Getter
 public class SeatResponse {
+    private long seatId;
     private String seatCode;
     private SeatStatus seatStatus;
 
-    @Override
-    public String toString() {
-        return "SeatResponse{" + "seatId='" + seatCode + '\'' + ", status=" + seatStatus + '}';
+    public SeatResponse(long seatId, SeatStatus seatStatus, String zoneName , long rowNumber, long colNumber) {
+        this.seatId = seatId;
+        this.seatStatus = seatStatus;
+        this.seatCode = zoneName + "-" + rowNumber + "-" + colNumber;
```

Comment:
seatCode는 response에서만 사용하고, request에서 얘를 받아서 뭘 하는 곳은 없는 것인가요?

```suggestion
        private static final String SEAT_CODE_FORMAT = "%s-%s-%s";
        this.seatCode = String.format(SEAT_CODE_FORMAT, zoneName, rowNumber, colNumber);
```

Reply:
사용자가 좌석을 선택하는 기능에서 해당 좌석의 상태를 체크할 때 사용하려고 합니다!